modules/ossm-vs-istio.adoc

```diff
@@ -100,6 +100,11 @@ spec:
 * Godebug has been removed from all templates
 * The `istio-multi` ServiceAccount and ClusterRoleBinding have been removed, as well as the `istio-reader` ClusterRole.
 
+[id="ossm-envoy-filters_{context}"]
+== Envoy filters
+
+{SMProductName} does not support `EnvoyFilter` configuration except where explicitly documented. This is due to tight coupling with the underlying Envoy APIs, meaning that backward compatibility cannot be maintained.  `EnvoyFilter` patches are very sensitive to the format of the Envoy configuration that is generated by Istio.  If the configuration generated by Istio changes, it has the potential to break the application of the `EnvoyFilter`.
```

Comment:
```suggestion
{SMProductName} does not support `EnvoyFilter` configuration except where explicitly documented. Due to tight coupling with the underlying Envoy APIs, backward compatibility cannot be maintained. `EnvoyFilter` patches are very sensitive to the format of the Envoy configuration that is generated by Istio. If the configuration generated by Istio changes, it has the potential to break the application of the `EnvoyFilter`.
```
- Consider tightening the wording of the second sentence. I've suggested one edit that might work.
- Change two spaces to one space between sentences.
- The phrase "`EnvoyFilter` patches are very sensitive to the format " strikes me as a bit too anthropomorphic. Consider rewording. (Also remove the "very".) I guess I'm not even sure what `very sensitive` means in this context, so maybe start with defining what exactly makes the patches have problems with the config formats.

Reply:
For that second sentence, how about:

Compatibility of EnvoyFilter patches between releases cannot be guaranteed, because they are tightly coupled to a specific version of configuration generated by Istio, which includes the structure of the configuration, as well as the Envoy API version being used.

IDK

Reply:
LOL, I saw just *right* after I pushed a change.  Now I have to think about this sentence again.

Reply:
I like the new version better than my suggestion.

Reply:
OK, I'll go with what I've got then.